benefits/oauth/urls.py

```diff
@@ -19,9 +19,9 @@
 urlpatterns = [
     # /oauth
     path(Routes.login, decorator(cdt_identity_views.login), kwargs, name=routes.name(routes.OAUTH_LOGIN)),
-    path(Routes.authorize, decorator(cdt_identity_views.authorize), kwargs, name=routes.name(routes.OAUTH_AUTHORIZE)),
-    path(Routes.cancel, decorator(cdt_identity_views.cancel), kwargs, name=routes.name(routes.OAUTH_CANCEL)),
+    path(Routes.authorize, decorator(cdt_identity_views.authorize), kwargs, name=Routes.authorize),
+    path(Routes.cancel, decorator(cdt_identity_views.cancel), kwargs, name=Routes.cancel),
     path(Routes.logout, decorator(cdt_identity_views.logout), kwargs, name=routes.name(routes.OAUTH_LOGOUT)),
```

Comment:
OK but can we just make all of these use `Routes.*` for the name (except `error`). There's no reason to mix them like this.

Reply:
Ah that's true, since both ways resolve to the same `name` string, might as well just use the simpler version. In 9769f6ffdf4eb6e8be121c1177f44e31749248c8 they now all use `Routes.*` for the `name` parameter except for the `error` url.